suites/apis/centralizedAuth.js

```diff
@@ -429,8 +434,8 @@ Scenario('User with access can CRUD indexd records in namespace, not outside nam
       abc_delete_response, new_abc_records.deleteMe,
       msg='user with permission could not delete record under `/abc`.'
     );
-    const getRes = await indexd.do.getFile(new_abc_records.deleteMe);
-    indexd.ask.recordDoesNotExist(getRes, new_abc_records.deleteMe);
+    const getRes = await indexd.do.getFileFullRes(new_abc_records.deleteMe);
+    indexd.ask.recordDoesNotExist(getRes);
```

Comment:
`getFileFullRes ` returns the raw response but `recordDoesNotExist` takes a `Gen3Response` as param and looks for a `statusCode` prop

Reply:
the `statusCode` property should exist on regular responses too. but I can cast to this Gen3 class